src/tools/clippy/clippy_utils/src/ty.rs

```diff
@@ -144,22 +146,26 @@ pub fn is_must_use_ty<'tcx>(cx: &LateContext<'tcx>, ty: Ty<'tcx>) -> bool {
     match ty.kind() {
         ty::Adt(adt, _) => must_use_attr(cx.tcx.get_attrs(adt.did)).is_some(),
         ty::Foreign(ref did) => must_use_attr(cx.tcx.get_attrs(*did)).is_some(),
-        ty::Slice(ty) | ty::Array(ty, _) | ty::RawPtr(ty::TypeAndMut { ty, .. }) | ty::Ref(_, ty, _) => {
```

Comment:
these unrelated formatting changes are probably due to an overeager rustfmt using rustc's rustfmt.toml instead of clippy's. This is likely going to mess with CI or a sync, not sure.

Reply:
we should really consolidate on one rustfmt.toml

Reply:
but I reverted them